### PR TITLE
gr-satellites: update version to 1.8.2

### PR DESCRIPTION
**Pull request by ra1nb0w**
#### Description

- bump version to 1.8.2
- fix livecheck
- patch already upstream

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.15.1 19B88
Xcode 11.1 11A1027

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: science/gr-satellites/Portfile [hunk 1]
```diff
@@ -11,16 +11,12 @@ maintainers         {@ra1nb0w irh.it:rainbow} {michaelld @michaelld} openmaintai
 description         GNU Radio decoders for several Amateur satellites
```
Comment (michaelld) [inline science/gr-satellites/Portfile]: Would you please add to the description (and a comment with the new livecheck) about that this port is for the 1.X branch? That way we're making it clear that this is not about the 2.Y or 3.Z branches. Thx!

Comment (ra1nb0w) [reply science/gr-satellites/Portfile]: is it ok now?
Anyway, my idea as I said through e-mail is to have gr37-satellites for GR37.

**Comment by ra1nb0w** [comment]
Can you commit https://github.com/macports/macports-ports/pull/5677 so that I update wxWidgets-3.2 branch? thanks

**Comment by michaelld** [comment]
> Can you commit #5677 so that I update wxWidgets-3.2 branch? thanks

Done

File: science/gr-satellites/Portfile (hunk 1 above)
Comment (michaelld) [reply science/gr-satellites/Portfile]: Yes yes I know ... still good to have the info for the interim ... thx!

**Comment by ra1nb0w** [comment]
thank you!

**Pull request merged**